ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -470,6 +470,12 @@ if [[ -f "${SHARED_DIR}/proxy-conf.sh" ]]; then
     source "${SHARED_DIR}/proxy-conf.sh"
 fi
 
+#Try installing jq in ansible image
+JQ_SPEC="https://api.github.com/repos/stedolan/jq/releases/latest" && \
```

Comment:
This should have been added to the image so downloading jq is no longer required.

Reply:
Ack. Also we need to remove existing code for JQ download which is in line 493 through 498.

Reply:
@jhou1 this is just to test with pj-rehearse. i will remove this part once ansible is ready with our verification-tests image.